src/plugincode/format_csv.py

```diff
@@ -0,0 +1,232 @@
+#
```

Comment:
IMHO there is a distinction here in `formattedcode` that provides plugins  and `formats.py` that provides the plugin mechanism and is core.
Eventually `formattedcode` could become a separate projects. Actual plugins contributed as a seperate packages could contribute, but in all cases the definition of the extension mechanism stays in the core. So it should not move to `formattedcode`